.expeditor/scripts/verify/run_clippy.sh

```diff
@@ -5,11 +5,11 @@ set -euo pipefail
 source .expeditor/scripts/verify/shared.sh
 
 toolchain="${1:-"$(get_toolchain)"}"
-install_rustup
-install_rust_toolchain "$toolchain"
+#install_rustup
```

Comment:
I assume we will get these uncommented...

Reply:
Yep, that should have never been committed